cmd/otel-agent/command/command.go

```diff
@@ -11,8 +11,11 @@ package command
 import (
 	"errors"
 	"flag"
+	"os"
 	"strings"
+	"time"
 
+	"github.com/kouhin/envflag"
```

Comment:
Do we need to bring a third party library to accomplish this functionality? I think it might be a bit overkill

Reply:
It's a small library, the code I'd have to write to actually integrate the env vars, with `flags` and `flags.flagset` would be almost as voluminous as the library itself... If you have a strong reason to push back given the above, I can write the code, but I'm not sure it's a big gain. Also, I would hope that we could eventually get rid of all of this and just leverage cobra, we can't do that at the moment.

Reply:
That makes sense. 

> Also, I would hope that we could eventually get rid of all of this and just leverage cobra, we can't do that at the moment.

That would be great. We could probably add a TODO or create a JIRA card so we are sure to remove it once we no longer needed

Reply:
One more question: Is this library added to all the other agents even if they don't use it?